client/landing/gutenboarding/onboarding-block/create-site/index.tsx

```diff
@@ -13,19 +13,41 @@ import CreateAndRedirect from './create-and-redirect';
 import { useNewQueryParam } from '../../path';
 import './style.scss';
 
+// Total time to perform "loading"
+const DURATION_IN_MS = 12000;
```

Comment:
Are we still actually finishing up when done any API juggling, even if the animation hasn't finished? :-)

Reply:
I don't think we _know_ anything about site creation progress, this is all UI.

Reply:
The header handles these effects, adding a domain to cart, creating a site, redirecting. Most of it is in the site store, but I don't think we have any insight into what's actually happening and how long it might take.

Reply:
I didn't mean we should know the duration, nor what's exactly happening, just that we do get signal when it's done and continue to next steps without waiting for all animation steps to finish. It might've been too obvious to point out but wanted to make sure. :-)

Reply:
Right, the animation and steps are _completely disconnected_ from any actual changes. It's all timing based.

That's not any different in this PR.